apis/goharbor.io/v1beta1/chartmuseum_types.go

```diff
@@ -61,6 +61,9 @@ type ChartMuseumSpec struct {
 
 	// +kubebuilder:validation:Required
 	Chart ChartMuseumChartSpec `json:"chart"`
+
+	// +kubebuilder:validation:Optional
+	Network *harbormetav1.Network `json:"network,omitempty"`
```

Comment:
As all harbor components need that field, why not put it into the `harbormetav1.ComponentSpec `json:"inline"`

Reply:
The Harbor type includes a HarborComponentsSpec. If the network is added to harbormetav1.ComponentSpec, there will be many networks in the Harbor type. It's confusing.